api/api_ethereum.go

```diff
@@ -770,6 +771,9 @@ type ethTxJSON struct {
 	ChainID    *hexutil.Big      `json:"chainId,omitempty"`
 	AccessList *types.AccessList `json:"accessList,omitempty"`
 
+	// Authorization list transaction (SetCodeTx) fields:
```

Comment:
```suggestion
	// SetCode transaction fields:
```

Reply:
@tnasu cc @blukat29 
If we follow convention it would probably look something like this, but what do you think?
```
	// Set code transaction fields:
```

Reply:
Yes that looks good!